### PR TITLE
Generate architecture SVG via API

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
This is to be able to generate an SVG file with architecture with the URLs using actual, deployed git hashes.

### CODE REVIEW

File: packit_dashboard/api/routes.py [hunk 1]
```diff
@@ -119,3 +125,26 @@ def usage_past_year():
     now = datetime.now()
     past_year_date = now.replace(year=now.year - 1).strftime("%Y-%m-%d")
     return _get_usage_data_from_packit_api(usage_from=past_year_date)
+
+
+@api.route("/api/images/architecture.svg")
```
[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply packit_dashboard/api/routes.py:130]: We can, but I want to use this image in the documentation.

But I am really not against doing it in a nicer way..;-) This was just the first thing that I was able to quickly implement.

[the remark this answers is not in the document]
Comment (mfocko) [reply packit_dashboard/api/routes.py:130]: What about this:
[](https://mermaid.live/edit#pako:eNqVU0tqwzAQvYoQXSYX8KJQyKYQaBovurBDkeWxLWxL7khOCMbQQ_SEPUklu_4lDaRaiNHM0xvp6amhXMVAPZoU6sQzhoZs96Ekdug6SpFVGXmpQPqZSEyfdyMWCNwIJTv0LM90FimGcbAZosNUZZUIdoznwhAf8Cg4kKfd8wxwUpgD6gH01i0P8wbjoWJm2JTuekfBTmmTIviv264eMQ2HJQghFjrYu5l81FDP6yDjP1txhbBkUSkGIbUz-f78Iqk9aqIwBVKICBmeQ3rRVVfAE1FAMAQD8nb3UUmyXj865RYydsmlBL_aXcPnhdtb-ktONV4Ins9OEdKHppnW7wYBSNuG1Fb2tZRCplbco9DOFCohJoMJHdJL4qF1T1t1730X58Ia17xOmwWn7o32H-5fb16TuwfvyW10F6PbMZrikm40Q885LO8ivjTSwG5dRFe0BCyZiO23blw6pHZLaS_k2TBmmDt0a3GsNso_S049gzWsaF1Zg8BGMGv8knoJKzS0PxiMUwA)

and then just jinja template the links

File: packit_dashboard/api/routes.py [hunk 2]
```diff
@@ -121,3 +131,16 @@ def usage_past_year():
...
+def architecture_diagram():
+    system_info = make_response(f"{API_URL}/system").json
+    system_info["packit_dashboard"] = dict(commit=os.environ.get("VITE_GIT_SHA"))
+    system_info["packit_worker"] = system_info["packit_service"]
```
Comment (mfocko) [inline packit_dashboard/api/routes.py:140]: Are you sure about this? :thinking: I don't think they're the same

**Review by mfocko — APPROVED** [review]
Apart from the worker v. service remark, LGTM

File: packit_dashboard/api/routes.py (hunk 2 above)
Comment (lachmanfrantisek) [reply packit_dashboard/api/routes.py:140]: We've discussed this locally. Currently, service and workers share repo and are built together so it's fine to expect this.

**Pull request merged**